markdown/boomi-v2-getting-started/boomi-v2-getting-started.md

```diff
@@ -278,35 +263,37 @@ Positive
 : Dont forget to "Save and Close" when done!
 
 
-### B. Configure Operation
-Back to the Start Shape configuration, click on the + icon for the **Operation** section to configure the operation parameters as seen in the screenshot below  
+### 4. Configure Operation
+Back to the Solace Shape configuration, click on the + icon for the **Operation** section to configure the operation parameters as seen in the screenshot below  
 
-`Mode`: `Persistent Transacted`     
-`Create Queue and TS`: `Selected`
+👉 Assign the Mode to Persistent Transacted    
+👉 Select the Create Queue and Topic Subscription option
 
 ![connectoroperation](img/connectoroperation.png "connectoroperation")
 
 Positive
-: Create Queue and TS means that Boomi will create the queue and subscription for you behind the scenes
+: Note: The Create Queue and Topic Subscription option means that Boomi will create the queue and subscription for you behind the scenes
```

Comment:
it'd be more accurate to say "... means that the Solace connector will create the queue and subscription for you..."